plugins/python_3_script/help.md

```diff
@@ -1,6 +1,6 @@
 # Description
 
-[Python](https://www.python.org/) is a programming language that lets you work quickly and integrate systems more effectively. This plugin allows you to run Python 3 code. It includes Python 3.6.5 and its standard library as well as the following 3rd party libraries: 
+[Python](https://www.python.org/) is a programming language that lets you work quickly and integrate systems more effectively. This plugin allows you to run Python 3 code. It includes Python 3.8 and its standard library as well as the following 3rd party libraries: 
 
 * [requests 2.18.4](https://requests.readthedocs.io/en/master/)
```

Comment:
not part of the pr but would it also be possible to bump these to

requests 2.31.0 (as this is the version of requests that comes back from pip list | grep requests on the rapid7/insightconnect-python-3-38-plugin:5 image)
maya 0.6.1 (as this is the version that is pinned in the requirements.txt file)
lxml 4.9.2 (as this is the version that is pinned in the requirements.txt file)
beautifulsoup 4.12.2 (as this is the version that is pinned in the requirements.txt file)
records 0.5.3 (as this is the version that is pinned in the requirements.txt file)